lib/project_types/theme/messages/messages.rb

```diff
@@ -53,6 +53,27 @@ module Messages
             APP
           },
         },
+        generate: {
+          env: {
+            ask_password_default: "Password (defaults to {{green:%s}})",
+            ask_password: "Password",
+            ask_store_default: "Store (defaults to {{green:%s}})",
+            ask_store: "Store",
+            ask_theme: "Select theme",
+            help: <<~HELP,
+              Generate a new env in {{green:config.yml}}.
```

Comment:
We may not want to say `generate a new env`, I'm not sure if that's immediately clear to everyone.
```suggestion
              Set the environment variables for this theme in {{green:config.yml}}.
```
Take the suggestion with a grain of salt 😅 .